readme.md

```diff
@@ -86,10 +86,10 @@
 ## Programming Languages
 
 - [JavaScript](https://github.com/sorrycc/awesome-javascript)
-	- [Promises](https://github.com/wbinnssmith/awesome-promises)
+	- [Promises](https://github.com/wbinnssmith/awesome-promises) - Resources for JavaScript Promises.
```

Comment:
This says nothing new. Think about it. Awesome lists are lists of resources. The name is `Promises` and it's a sub-item of `JavaScript`.

Also note:

> Include a short description about the project/theme of the list. It should not describe the list itself. - https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md

Same applies to all the other descriptions here.